CI/Import/Functions.sh

```diff
@@ -8,10 +8,10 @@ function get_changed_files() {
 
   if [[ -z ${PR_NUMBER} ]]
   then
-    CHANGED_FILES=$(git diff-tree --name-only --diff-filter=ACMRT --no-commit-id -r ${GH_SHA} | grep '.php')
+    CHANGED_FILES=$(git diff-tree --name-only --diff-filter=ACMRT --no-commit-id -r ${GH_SHA} | grep -e '.php$' -e '.js$')
   else
     URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}/files"
-    CHANGED_FILES=$(curl -s -X GET -G $URL | jq -r '.[] | .filename' | grep '.php')
+    CHANGED_FILES=$(curl -s -X GET -G $URL | jq -r '.[] | .filename' | grep -e '.php$' -e '.js$')
```

Comment:
The regular expressions in both `grep` calls need escaping of the period character. I suggest to use `grep -e '\.php$' -e '\.js$'` instead.

(This used to be wrong before, but I’d suggest to fix it while we’re at it.)